ReSharper.FSharp/test/data/features/service/arguments/Unit 01 - Named.fs

```diff
@@ -0,0 +1,4 @@
+type T() =
+    static member M(u: unit) = ()
+
+{selstart}T.M(u = ()){selend}
```

Comment:
Thanks for adding the test case! If we're to change it later, we'll have to update the recorded behavior explicitly, which is good.